experimental/decimal.md

```diff
@@ -0,0 +1,25 @@
+# [Decimal][proposal-decimal]
+
+# Literal
+
+```js
+extend interface Literal <: Expression {
+  type: "Literal";
+  value: string | boolean | null | number | RegExp | bigint | bigdecimal
+}
+```
+
+- `value` property can be a `bigdecimal` value to represent decimal literals, e.g. `2.718m`.
+
+## DecimalLiteral
```

Comment:
In ESTree we haven't had separate nodes for different types of literals. I know Babel does, but maybe it should be left out here?

Reply:
Note that the `DecimalLiteral` does not have a "DecimalLiteral" type property. For `0.3m`, it has an extra `decimal: string` property that stores the string representation `0.3`.

We have similar settings for `BigIntLiteral`: https://github.com/estree/estree/blob/master/es2020.md#bigintliteral

Reply:
Oh I see. Thanks for explaining.